embedded-login/src/main.js

```diff
@@ -78,14 +78,19 @@ const getStage = (step) => {
 const handleStep = async (step) => {
   switch (step.type) {
     case 'LoginSuccess': {
-      // If we have a session token, get user information
-
-      // eslint-disable-next-line no-unused-vars
-      const sessionToken = step.getSessionToken();
-      // eslint-disable-next-line no-unused-vars
-      const tokens = await forgerock.TokenManager.getTokens();
-      const user = await forgerock.UserManager.getCurrentUser();
-      return showUser(user);
+      if (goto != null) {
+        window.location.replace(goto);
+        return;
+      } else {
```

Comment:
I'd honestly rather have a dedicated app for this, rather than add it as a feature of the existing "embedded-login". This may confuse people and find it a bit harder to discover.

Reply:
I would like that too but we don't have that at the moment. From my experience mostly customers ask how to use an "Embedded Login" app to do Centralized Login. So on the other hand when building a Self-Hosted app this might be the best approach, to have an app that can be used both directly and as a "Custom Login" app.

Reply:
In the absence of another example we can point the customers at the moment I would suggest we have that in and we can revise in the future